src/core/server.js

```diff
@@ -913,7 +918,7 @@ Server.prototype._reserve = function(ownerCount) {
     throw new Error('Owner count must not be negative.');
   }
 
-  return reserve_base.add(reserve_inc.product_human(owner_count));
+  return reserve_base.add(reserve_inc.multiply(owner_count));
```

Comment:
product_human, once upon a time had special casing for integer Numbers, coercing them into strings, then tacking a '.0' on the end, so the amount would be parsed as XRP rather than drops.